Embedded Software/TelemetryControlUnit/lib/WDT_T4-master/Watchdog_t4.h

```diff
@@ -0,0 +1,66 @@
+#if !defined(_WATCHDOG_T4_H_)
```

Comment:
Not sure how legal it is to just copy-paste another repo into this, that's why I made the WDT_T4 a submodule so it was just a dependency, but submodules are dumb and we are just a club so like it probably doesn't matter